src/run.js

```diff
@@ -109,7 +110,14 @@ async function write (changelog, options, log) {
   const bytes = Buffer.byteLength(changelog, 'utf8')
   const existing = await fileExists(options.output) && await readFile(options.output, 'utf8')
   if (existing) {
-    const index = existing.indexOf(PREPEND_TOKEN)
+    let index = existing.indexOf(PREPEND_TOKEN)
+
+    if (index === -1) {
+      if(options.prepend) {
+        index = 0;
+      }
+    }
```

Comment:
```suggestion
    if (index === -1 && options.prepend) {
      index = 0;
    }
```

Reply:
lol, yea 😂 completely right! Maybe my brain was having a day off yesterday and I didn't notice 🙃 🧠